### DISCUSSION TITLE
How can I set up QfieldCloud on my own PC?

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#4575] [Q&A]
Hello everyone, I'm completely new to Qfield and QfieldCloud. I've been tasked with setting up a QField Cloud instance on our company's server.

I lack experience in IT, and although I've come across the QfieldCloud repository, I'm unsure about how to proceed. I've already tried the README file on the repository of QField Cloud. I've already tried the README file on the repository for QField Cloud. but I have an error: 'git' is not recognised as an internal or external command.Any assistance or guidance would be greatly appreciated. when i clone the first step.
Any assistance or guidance would be greatly appreciated.

**Comment by PierreSerpe** [comment]
Hi did you get further on the installation?

**Reply by mahmoodi-source** [reply]
Hi, our IT team built a could server, and I have no idea how? sorry.
